api/src/opentrons/protocol_api/module_contexts.py

```diff
@@ -974,3 +974,49 @@ def deactivate_shaker(self) -> None:
     def deactivate_heater(self) -> None:
         """Stop heating."""
         self._module.deactivate_heater()
+
+    def flag_unsafe_move(
+        self,
+        to_loc: types.Location,
+        is_multichannel: bool,
+    ) -> None:
+        """
+        Raise an error if attempting to perform a move that's deemed unsafe due to
+        the presence of the heater-shaker.
+        """
+        is_labware_latch_closed = (
+            self._module.labware_latch_status
+            is HeaterShakerLabwareLatchStatus.IDLE_CLOSED
```

Comment:
```suggestion
            self._module.labware_latch_status
            == HeaterShakerLabwareLatchStatus.IDLE_CLOSED
```

I don't think it matters in this case because `self._module.labware_latch_status` *will* return the exact `HeaterShakerLabwareLatchStatus.IDLE_CLOSED` object, but as a general rule, I think we should avoid `is` unless we *really* want to compare object identity, specifically, as opposed to object value.